src/pages/Regions/Region.tsx

```diff
@@ -838,7 +838,7 @@ const Region = () => {
                           />
                         </Tooltip>
                       </Col>
-                      {activeUserRoles.includes(Roles.Admin) ? (
+                        {(activeUserRoles.includes(Roles.Admin) || activeUserRoles.includes(Roles.GoverningBodyAdmin)) ? (
```

Comment:
I suggest to move this to a json file on a backend and give back dictionary with permissions, beacause we  might need to add new roles in a future and it will become unmaneagable.